testdata/openapi2/src/interface/want.yaml

```diff
@@ -0,0 +1,34 @@
+swagger: "2.0"
+info:
+  title: x
+  version: x
+consumes:
+- application/json
+produces:
+- application/json
+paths:
+  /path:
+    get:
+      operationId: GET_path
+      produces:
+      - application/json
+      responses:
+        200:
+          description: 200 OK
+          schema:
+            $ref: '#/definitions/interface.resp'
+definitions:
+  interface.fooer:
+    title: fooer
+    description: fooer is something.
+    type: object
```

Comment:
The generated docs for this are kind of ... not so useful: https://app.swaggerhub.com/apis/Teamwork/teamwork-desk/2.0

![2018-08-13-155241_348x108_scrot](https://user-images.githubusercontent.com/1032692/44039403-eb3e3020-9f10-11e8-907b-c94946a2135b.png)

I don't know if there is a good way to improve on that though.